etl/src/v2/workers/apply.rs

```diff
@@ -396,27 +346,23 @@ where
     }
 
     async fn process_syncing_tables(&self, current_lsn: PgLsn) -> Result<bool, Self::Error> {
-        let table_replication_states = self.active_table_replication_states().await?;
+        let table_replication_states = self.active_table_replication_states(false).await?;
```

Comment:
We need to be very careful with this, since it might backfire in case we add data to the db and not in memory.

Reply:
The `active_table_replication_states` method no longer takes a boolean after the new state store rework.